src/cluecode/finder_data.py

```diff
@@ -200,18 +217,24 @@ def set_from_text(text):
    .png
    .jpg
    .gif
+   .jpeg
 ''')))
 
 
-def classify(s, data_set, suffixes=None):
+def classify(s, data_set, suffixes=None, ignored_hosts=None):
```

Comment:
Can you add tests for this method since you changed it?

Reply:
I've added a few tests here: https://github.com/nexB/scancode-toolkit/pull/1882/files#diff-b64a4b3f81b0ea3294cba2a968debaa5R185
I'll add a few more with the next push!